readme.md

```diff
@@ -160,6 +160,7 @@ If you are new to eBPF, you may want to try the links described as "introduction
 
 ## Tutorials
 
+- [Building BPF applications with libbpf-bootstrap](https://nakryiko.com/posts/libbpf-bootstrap/) - Get started with your own BPF application quickly and painlessly using the modern BPF CO-RE facilities. No runtime clang/LLVM dependency whatsoever.
```

Comment:
It is a dense read, quite not addressed to people unfamiliar with eBPF, so I have mixed feelings about having it in first position. At the same time I understand it is better to orient people towards this guide rather than the Python/libllvm machinery, so why not. We are desperately missing an easy tutorial on how to get started from scratch with tracing eBPF program :/.

The description itself can maybe be improved. CO-RE is important indeed, but “get started [...] quickly and painlessly” is pretty much the objective of any tutorial, and CO-RE will only be “modern” for so long. The note on clang/LLVM is not super informative either, I understand this is by contrast with BCC, but then the examples for most of the other resources in this section have no runtime dependency on LLVM either.

What about something like:
```
Helps generate minimal or advanced templates to bootstrap your own applications (kernel side and user space management for maps and programs) with features like CO-RE, global variables, and ring buffer.
```

Reply:
Yep, I was also torn on putting it in the first position. However, after reading the [deprecation news](http://www.brendangregg.com/blog/2020-11-04/bpf-co-re-btf-libbpf.html) I decided to put it at the beginning of the list. However, I don't have a strong option about that.

The LLVM part of the description was mostly an attempt to emphasize the main difference of libbpf CO-RE approach from the more "traditional" BCC style where clang/LLVM are runtime dependencies. Although, BCC seems to be moving to libbpf and CO-RE too. To be honest, I didn't put much effort into the description and I'm absolutely fine with your version.